modules/ROOT/pages/dataweave-pattern-matching.adoc

```diff
@@ -97,9 +97,9 @@ output application/json
 }
 ----
 
-For use cases where you need a boolean result based on whether a value
-matches or not, you can use the `matches` function instead of using pattern
-matching.
+For use cases in which the input is of type String, you can also use the xref:dw-core-functions-match.adoc[match] _function_ in the `dw::Core` module to test the string against a regex pattern. The function returns an array with the string that matches the entire regex and its capture groups.
```

Comment:
If the function returns an array shouldn't it be strings in plural? _The function returns an array with the string**s** that match the entire regex and its capture groups._, unless it returns an array with just one element.

Reply:
Yep, looking at the second match example on docs.mulesoft.com, it returns matching strings (plural) in an array. Fixing.